Library/Homebrew/cmd/audit.rb

```diff
@@ -411,7 +411,7 @@ def audit_desc
       problem "Description shouldn't start with an indefinite article (#{$1})"
     end
 
-    if desc =~ /^#{formula.name}\s/i
+    if desc =~ /^#{Regexp.escape formula.name}\s/i
```

Comment:
Yeah, I’ll merge that one quickly to fix #48744 and will look at the others just after.